scripts/pages/settings/fov.js

```diff
@@ -0,0 +1,58 @@
+class Fov {
+	static panels = {
+		/** @type {SettingsSlider} @static */
+		fov: $('#FOV'),
+		/** @type {TextEntry} @static */
+		horizontalFov: $('#FOV_Horizontal')
+	};
+
+	static loadSettings() {
+		this.updateFOV();
+	}
+
+	static aspectRatio() {
+		// find the display panel
+		// there doesn't seem to be an api for this yet
+		let panel = this.panels.fov;
+		let parent = panel;
+		while ((parent = panel.GetParent())) {
+			panel = parent;
+		}
+
+		return panel.actuallayoutwidth / panel.actuallayoutheight;
```

Comment:
I don't mind doing this, but I'd rather we use it to pick a default aspect ratio, then show alternatives in a dropdown. Should also having a info message on the RHS if we're doing this, let's discuss wording in Discord.